### PR TITLE
global: Invenio 3 compatibility

### PR DESCRIPTION
**Pull request by jalavik**
* INCOMPATIBLE Changes module to be compatible with Invenio 3.

Signed-off-by: Jan Aage Lavik <jan.age.lavik@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,26 @@
 # -*- coding: utf-8 -*-
 #
-# This file is part of Invenio
-# Copyright (C) 2015, 2016 CERN
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
```
Comment (jirikuncar) [inline .travis.yml]: it's good practice to keep past copyright years

File: invenio_classifier/cli.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        output_limit=output_limit,
+        spires=spires,
+        match_mode=match_mode,
+        no_cache=no_cache,
+        with_author_keywords=detect_author_keywords,
+        rebuild_cache=rebuild_cache,
+        only_core_tags=only_core_tags,
+        extract_acronyms=extract_acronyms
+    )
+    print(result)
```
Comment (jirikuncar) [inline invenio_classifier/cli.py]: better use `click.echo` instead of `print`

File: .travis.yml (hunk 1 above)
Comment (jalavik) [inline .travis.yml]: Yes, indeed, my bad, copy mistake.

File: invenio_classifier/cli.py (hunk 2 above)
Comment (jalavik) [inline invenio_classifier/cli.py]: Right! Thanks.

**Comment by jalavik** [comment]
Yay, :ship: ?

**Pull request merged**